adapters/LiftoffMonetize/LiftoffMonetizeAdapter/GADMediationAdapterVungle.h

```diff
@@ -18,17 +18,8 @@
 typedef NS_ENUM(NSInteger, GADMAdapterVungleErrorCode) {
```

Comment:
The other error codes are handled within the SDK. The two remaining ones issues when retrieving valid information or object from the adapter from the Google SDK